### ISSUE TITLE
单卡GPU配置推荐

### ISSUE THREAD

**Issue by zhjunqin** [PaddlePaddle/PaddleClas#2589]
在使用单卡训练的时候会有如下 warning，默认的配置是 4 卡 GPU 的，那么如果使用单卡的话，应该如何修改配置才比较好呢？

WARNING: The training strategy provided by PaddleClas is based on 4 gpus. But the number of gpu is 1 in current training. Please modify the stategy (learning rate, batch size and so on) if use this config to train.

能否以 https://github.com/PaddlePaddle/PaddleClas/blob/release/2.5/ppcls/configs/PULC/person_attribute/PPLCNet_x1_0.yaml 为例子，帮忙解释一下，多谢

**Comment by zh-hike** [comment]
> 

4卡变单卡学习率除以4就可以了
